spyder/plugins/updatemanager/widgets/update.py

```diff
@@ -26,11 +27,15 @@
 from spyder.api.translations import _
 from spyder.config.base import is_conda_based_app
```

Comment:
```suggestion
from spyder.config.base import is_conda_based_app
from spyder.config.gui import is_dark_interface
```

To go with my suggestion about setting the right value for `interface_theme` (see below).